src/main/java/uk/gov/hmcts/reform/pip/account/management/controllers/AccountController.java

```diff
@@ -74,6 +74,24 @@ public ResponseEntity<Map<CreationEnum, List<?>>> createUsers(//NOSONAR
         return ResponseEntity.status(HttpStatus.CREATED).body(accountService.addUsers(users, issuerId));
     }
 
+    /**
```

Comment:
Are we able to put the PreAuthorise on the existing endpoint instead? I'm thinking the only place that should be creating users is our frontend which should be passing in an issuer ID so I don't think is a breaking change

Reply:
That makes sense. I have removed the new v2 endpoint for create PI user